frontend/__tests__/src/components/metrics/ConfigStep/MetricsTypeCheckbox.test.tsx

```diff
@@ -77,4 +77,24 @@ describe('MetricsTypeCheckbox', () => {
 
     expect(queryByText(CONFIG_TITLE.BOARD)).toBeNull()
   })
+
+  it('should pipelineTool component when click MetricsTypeCheckbox selection Lead time for changes ', () => {
+    const { getByRole } = setup()
+    fireEvent.mouseDown(getByRole('button', { name: REQUIRED_DATA }))
```

Comment:
I don't think the `mouseDown` is a really user entire user behavior. I think it doesn't make sense to tell from the `mouseDown` and `click` from a users perspective. 
Compared with the `fireEvent`, `userEvent` could simulate user behavior better. For exemple, `userEvent` will contain all the mouseDown, click, mouseUp...  fireEvent being the low-level api, while userEvent sets a flow of actions. I recommend to use `userEvent` in the test.

Reply:
Yep, agree with that, will use `userEvent` instead of `fireEvent` for all tests.👍